src/courses/advanced/05.md

```diff
@@ -408,19 +417,41 @@ class Git < Inspec.resource(1)
     end
 
     def branches
-        inspec.command("git --git-dir #{@path} branch").stdout
+        inspec.command("git --git-dir #{@path} branch --format='%(refname:short)'").stdout.lines.map(&:strip)
     end
 
     def current_branch
-        branch_name = inspec.command("git --git-dir #{@path} branch").stdout.strip.split("\n").find do |name|
-            name.start_with?('*')
-        end
-        branch_name.gsub(/^\*/,'').strip
+        inspec.command("git --git-dir #{@path} branch --show-current").stdout.strip
     end
 
 end
 ```
 
+::: tip 1337 h4x0rs
```

Comment:
I think this 'hack code' may be confusing to students

Reply:
it'll probably be fine :)

if they don't get it, they can ignore it or use google lol

if they do get it, then they'll get a small chuckle